keycloak/themes/uid2-theme/email/messages/messages_en.properties

```diff
@@ -1,8 +1,8 @@
 # encoding: UTF-8
-emailVerificationBodyHtml=<h2 style="font-weight: 700; font-size: 22px; line-height: 21px"> Please verify your email address </h2> <div style="text-align: center"> <p style=" font-size: 18px; color: #000; font-family: 'Inter', sans-serif; max-width: 450px; margin: 0 auto; " > Click the button to verify this email. We’ll review your account and be in touch within the next few business days. </p> </div> <div style="margin-top: 40px"> <a href="{0}" style=" border-radius: 5px; font-weight: 700; font-size: 18px; padding: 10px 42px; margin: 0 auto; background-color: #cdf200; color: #030a40; width: auto; text-align: center; text-decoration: none; " >Verify Email</a > </div>
+emailVerificationBodyHtml=<h1> Please verify your email address </h1> <p class="content-text" > Click the button to verify this email. We’ll review your account and be in touch within the next few business days. </p> <div class="footer"> <a href="{0}" class="button" >Verify Email</a > </div>
 emailVerificationBody=Please verify your email address\n\nClick the link below to verify this email and complete your account creation for Unified ID.\n\n{0}
 passwordResetSubject=Reset password
-passwordResetBodyHtml=<div style="text-align: center"> <p style="text-align: left"> Hey {0},<br /><br /> Forgot your Password?<br /><br /> We received a request to reset your password for your account.<br /><br /> Please use the button below to reset your password. </p> </div> <div style="margin: 66px 0"> <a href="{1}" style=" border-radius: 5px; font-weight: 700; font-size: 18px; padding: 10px 23px; margin: 0 auto; background-color: #cdf200; color: #030a40; width: auto; text-align: center; text-decoration: none; " >Reset Password</a > </div>
+passwordResetBodyHtml=<p> Hey {0}, </p> <p> Forgot your Password? </p> <p> We received a request to reset your password for your account. </p> <p> Please use the button below to reset your password. </p> <div class="footer"> <a href="{1}" class="button" >Reset Password</a > </div>
```

Comment:
I know this copy isn't new but the capitalised "Password" looks odd to me. What about just `Forgot your password?`

Reply:
I checked the designs and looks like Kristen has removed that line altogether - so it's gone now :)